README.md

```diff
@@ -180,7 +180,17 @@ const model = wrapLanguageModel({
 
 ## Model Context Protocol
 
-The Stripe Agent Toolkit also supports the [Model Context Protocol (MCP)](https://modelcontextprotocol.com/). See `/typescript/examples/modelcontextprotocol` for an example. The same configuration options are available, and the server can be run with all supported transports.
+The Stripe Agent Toolkit also supports the [Model Context Protocol (MCP)](https://modelcontextprotocol.com/).
+
+To run the Stripe MCP server using npx, use the following command:
+
+```bash
+npx -y @stripe/mcp --tools=all --api-key=<YOUR-STRIPE-SECRET-KEY>
```

Comment:
small thing: i would replace this, because then it is easy for developers to double click 'your_stripe_secret_key' and replace all of it

```suggestion
npx -y @stripe/mcp --tools=all --api-key=YOUR_STRIPE_SECRET_KEY
```